config_src/drivers/nuopc_cap/mom_cap.F90

```diff
@@ -648,6 +649,13 @@ subroutine InitializeAdvertise(gcomp, importState, exportState, clock, rc)
 
   endif
 
+  ! Read end of run restart config option
```

Comment:
I think the retrieval of the config parameter is more naturally located in the InitializeP0 routine,  where the other config files are read.